### PR TITLE
Add a 'help' button to the toolbar, opening a configurable url in a n…

### PR DESCRIPTION
**Pull request by landryb**
…ew window

The help button reuses the info icon from mapfishapp.

By default https://github.com/georchestra/cadastrapp/wiki/Guide-Utilisateur is
used, but options.helpUrl can be set in config.json.

### CODE REVIEW

**Comment by landryb** [comment]
Should fix #217

File: addons/cadastrapp/README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Il faut rajouter ```ROLE_``` devant le nom du groupe ldap. Par exemple si votre
 		"de": "TODO" },
 	"options": { 
 		"target": "tbar_12",
+		"helpUrl": "https://github.com/georchestra/cadastrapp/wiki/Guide-Utilisateur",
```
Comment (fvanderbiest) [inline addons/cadastrapp/README.md:35]: rather in manifest.json by default, overridable here ...

File: addons/cadastrapp/js/main.js [hunk 2]
```diff
@@ -42,6 +42,7 @@ GEOR.Addons.Cadastrapp = Ext.extend(GEOR.Addons.Base, {
                           
                 GEOR.Addons.Cadastre.menu = new GEOR.Addons.Cadastre.Menu({
                     map: initThis.map,
+                    helpUrl: record.data.options.helpUrl,
```
Comment (fvanderbiest) [inline addons/cadastrapp/js/main.js:45]: this.options.helpUrl ?

**Comment by pierrejego** [comment]
Voir pour ajouter une icône aide

File: addons/cadastrapp/js/main.js (hunk 2 above)
Comment (landryb) [inline addons/cadastrapp/js/main.js:45]: Tried it, and fails with `TypeError: this.options is undefined` in the console..... so yeah, scopes :(

File: addons/cadastrapp/README.md (hunk 1 above)
Comment (landryb) [inline addons/cadastrapp/README.md:35]: 100% agreed, will update/merge once #225 is merged

**Pull request merged**